components/tidb_query_datatype/src/codec/data_type/mod.rs

```diff
@@ -159,6 +159,20 @@ pub trait EvaluableRet: Clone + std::fmt::Debug + Send + Sync + 'static {
     fn into_vector_value(vec: Self::ChunkedType) -> VectorValue;
 }
 
+/// # Notes
+///
+/// Make sure operating `bitmap` and `value` together, so while `bitmap` is 0 and the
```

Comment:
Can we ensure it though Rust interface?

Reply:
Currently all functions in ChunkedVec ensures this property, and data inside ChunkedVec can't be accessed by other modules. This is just a note for future developers.